app/modules/import-export/assets/js/shared/process-failed-dialog/process-failed-dialog.js

```diff
@@ -60,6 +60,10 @@ const messagesContent = {
 		title: __( 'Couldn’t install the Kit', 'elementor' ),
 		text: __( 'The Kit includes plugins you don’t have permission to install. Contact your site admin to change your permissions.', 'elementor' ),
 	},
+	'no-domdocument': {
+		title: __( 'Couldn’t install the Kit', 'elementor' ),
```

Comment:
```suggestion
		title: __( "Couldn't install the Kit", 'elementor' ),
```

Reply:
Don`t you think it is better to keep the same way as the other errors?